### ISSUE TITLE
Ego vehicle centered

### ISSUE THREAD

**Issue by TheCodez** [TheCodez/dynamic-occupancy-grid-map#53]
![Webp net-gifmaker (6)](https://user-images.githubusercontent.com/4009589/80920755-27199e80-8d72-11ea-8d2e-4a527cd0ffe5.gif)

@cbachhuber this is what I was talking about in https://github.com/TheCodez/dynamic-occupancy-grid-map/pull/47

**Comment by cbachhuber** [comment]
Right, thanks for drawing :+1: 

I guess implementing this makes sense when the framework is flexible enough to support multiple sensor inputs. Currently, I don't see a large benefit of this. Do you agree with this?

Are you adhering to the priorities assigned to the issues? Now that the end-to-end correctness is proven, my highest priority in this repo is cleaning the architecture. That's at the moment splitting the quite large DOGM-godclass into smaller elements such as the measurement grid generator (your work on #8 and #51). Once that is done, I imagine separating the grid data from the functionality would make sense.

**Comment by TheCodez** [comment]
Agree, this is actually one of the reasons this still isn't implemented.

Yes, I'll reorder the labels based on the current priorities.
